specification/apimanagement/resource-manager/readme.md

```diff
@@ -1057,4 +1060,10 @@ directive:
   - suppress: R4009
     from: apimpolicyfragments.json
     reason: Warning raised to error while PR was being reviewed. SystemData will implement in next preview version. 
+suppressions:
+  - code: PropertiesTypeObjectNoDefinition
+    from: definitions.json
+    reason: Invalid error
```

Comment:
>son: Invalid error [](http://example.com/codeflow?start=7&length=18)

Can you please elaborate , why this is being suppressed ?

Reply:
This comment is not confirmed. Why this is being suppressed ?

Reply:
As per my Communication with Roopesh Manda and Tejaswi Salaigari, this rule was a false alarm and they recommended to suppress this error